justfile

```diff
@@ -0,0 +1,19 @@
+# Install with `cargo install just`
+# Usage: `just --dotenv-filename /path/to/file.env <bench|gpu-bench>`
+set dotenv-load
+
+print-lurk-env:
+  echo $LURK_PERF_CONFIG
+  echo $LURK_RC
+
+bench:
+  print-lurk-env
+  cargo criterion --bench fibonacci
+
+gpu-bench:
+  print-lurk-env
+  cargo criterion --bench fibonacci
+
+deploy-bench:
+  print-lurk-env
+  cargo criterion --bench fibonacci --message-format=json > ${{ github.sha }}.json
```

Comment:
This seems a bit jumbled. How about this?
```
cuda-bench:
  print-lurk-env
  EC_GPU_FRAMEWORK=cuda cargo criterion --bench fibonacci --features "cuda" --message-format=json | tee ${{ github.sha }}.json
```

(`EC_GPU_FRAMEWORK` could be in the bench.env)